core/src/main/java/com/google/zxing/ResultMetadataType.java

```diff
@@ -93,5 +93,15 @@
    * parity is given with it.
    */
   STRUCTURED_APPEND_PARITY,
-  
+
+  /**
+   * For Code39 this contains the decoded contents with the Full-ASCII range.
+   */
+  CODE_39_EXTENDED,
+
+  /**
+   * For Code39 this contains a boolean marking whether the checksum is valid.
+   */
+  CODE_39_CHECKSUM_CORRECT,
```

Comment:
I might be dreaming too far here, but since there are a few formats like this, what about `OPTIONAL_CHECKSUM_CORRECT` or something? no point promising it for required checksums but this might have a similar role for other formats.